### PR TITLE
Ya user profile

### PR DESCRIPTION
**Pull request by CodeX-y**

### CODE REVIEW

**Review by kcalva — APPROVED** [review]
Everything looks good. 
- managed to route to the correct path when adding a comment to a post.
- created a partial form (since its being used multiple times)
- created a route to display the users posts.
Great job

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        <div class="card-body">
+          <h5 class="card-title"><%= photo.owner.username %></h5>
+          <p class="card-text"><%= photo.caption %></p>
+        </div>
+        <ul class="list-group list-group-flush">
+          <% photo.comments.each do |comment| %>
+            <li class="list-group-item">
+              <div class="d-flex">
+                <div class="flex-shrink-0">
+                  <img src="..." alt="...">
```
Comment (JoseMDO) [inline app/views/users/show.html.erb:24]: Just small thing here, if you don't use an image tag its always good to add default values or just get rid of it all together :)

File: config/routes.rb [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
+
+  devise_for :users
   
   resources :likes
   resources :follow_requests
   resources :comments
   resources :photos
-  devise_for :users
+
+  get "/:username" => "users#show", as: :user
```
Comment (JoseMDO) [inline config/routes.rb:11]: Nice!

**Review by JoseMDO — APPROVED** [review]
Looks good Yusra!

File: app/controllers/comments_controller.rb
Comment (Cindellai) [inline app/controllers/comments_controller.rb:1]: Great job ensuring the comment.author is set to current_user.

File: app/views/users/show.html.erb
Comment (Cindellai) [inline app/views/users/show.html.erb:1]: Great job using partials to render the comments form

**Pull request merged**